docs/topics/api/signing.rst

```diff
@@ -64,6 +67,43 @@ validation and you will be able to check its status.
     :statuscode 403: you do not own this add-on.
     :statuscode 409: version already exists.
 
+
+Uploading without an ID
+-----------------------
+
+.. note::
+    This is only valid for `WebExtensions <https://wiki.mozilla.org/WebExtensions>`_.
+    All other addon types require an addon-id and have to use the regular
+    endpoint to :ref:`upload a version <upload-version>`.
+
+
+.. http:post:: /api/v3/addons/[string:version]/
```

Comment:
Since this is no longer a PUT, I feel like `version` should be a POST'ed parameter now. This way we could also make it optional, and parse it from the manifest instead. However, this could be done in a follow-up patch, no need to block on it.